tdrs-backend/tdpservice/parsers/util.py

```diff
@@ -13,6 +12,15 @@ def value_is_empty(value, length):
     return value is None or value in empty_values
 
 
+error_types = {
+    1: 'File pre-check',
+    2: 'Record value invalid',
+    3: 'Record value consistency',
+    4: 'Case consistency',
+    5: 'Section consistency',
+    6: 'Historical consistency'
+}
+
```

Comment:
I think this was prototype for the new parserError category `choices`. Should be removed I think?

Reply:
i've updated the 1113-2 and 1610 base branches to address this as well as the other linting errors.